src/sentry/utils/signing.py

```diff
@@ -12,13 +12,15 @@
 
 
 def sign(**kwargs):
-    return base64.urlsafe_b64encode(TimestampSigner(salt=SALT).sign(dumps(kwargs))).rstrip("=")
+    return base64.urlsafe_b64encode(
+        TimestampSigner(salt=SALT).sign(dumps(kwargs)).encode("utf-8")
```

Comment:
Does this actually provide the same result in p2 and py3? Django uses `force_str` a bunch here, which produces a bytestring in py2 and a unicode string in py3.

I don't see any tests that explicitly make sure this is the case... might be worth writing one to make sure 2/3 produce the same results.